setup.py

```diff
@@ -47,13 +47,19 @@
         "strawberryfields",
         "strawberryfields.circuitspecs",
         "strawberryfields.apps",
+        "strawberryfields.cli",
         "strawberryfields.backends",
         "strawberryfields.backends.tfbackend",
         "strawberryfields.backends.fockbackend",
         "strawberryfields.backends.gaussianbackend",
     ],
     "package_data": {"strawberryfields": ["backends/data/*", "apps/data/*"]},
     "include_package_data": True,
+    "entry_points" : {
```

Comment:
An approach using `entry_points` was chosen such that the CLI can be tested with ease. This allowed creating tests comparable to those of other components.